tests/test_query_params.py

```diff
@@ -136,6 +136,7 @@ def test_simple(self, simulate_request, client, resource):
     def test_percent_encoded(self, simulate_request, client, resource):
         query_string = 'id=23,42&q=%e8%b1%86+%e7%93%a3'
         client.app.add_route('/', resource)
+        client.app.req_options.auto_parse_qs_csv = True
```

Comment:
We have a couple of tests below setting *auto_parse_qs_csv* to False. I was thinking to myself if we could morph one of them, or add a new test, to check that the behaviour is corresponding to the new default even without setting it explicitly?

Reply:
@vytas7 Yeah, agree! I'll add one more test without setting it explicitly, also I think that I'll push a commit in 1-2~ weeks, it works for you?

Reply:
Sure that works, your effort is much appreciated!